src/v2/server.ts

```diff
@@ -35,70 +20,22 @@ app.get(
     }
 
     try {
-      const {
-        status,
-        styleTags,
-        scripts,
-        redirect,
-        bodyHTML,
-        headTags,
-      } = await buildServerApp({
+      const { status, redirect, ...rest } = await buildServerApp({
+        assetsPath: "/assets",
+        loadableFile: "loadable-novo-stats.json",
+        loadablePath: "public/assets",
+        next,
         req,
         res,
-        next,
         routes,
-        loadableFile: "loadable-novo-stats.json",
-        loadablePath: "public/assets",
-        assetsPath: "/assets",
       })
 
       if (redirect) {
         res.redirect(status ?? 302, redirect.url)
         return
       }
 
-      const headTagsString = ReactDOM.renderToString(headTags as any)
-      const sharify = res.locals.sharify
-
-      const { APP_URL, CURRENT_PATH, WEBFONT_URL } = sharify.data
-
-      const options = {
-        cdnUrl: NODE_ENV === "production" ? CDN_URL : "",
-        content: {
-          body: bodyHTML,
-          data: getV2SharifyScript(sharify.data),
-          head: headTagsString,
-          scripts,
-          style: styleTags,
-        },
-        disable: {
-          analytics: getServerParam(req, "disableAnalytics") === "true",
-          onetrust: getServerParam(req, "disableOneTrust") === "true",
-          postie: getServerParam(req, "disablePostie") === "true",
-          segment: getServerParam(req, "disableSegment") === "true",
-          stripe: getServerParam(req, "disableStripe") === "true",
-        },
-        env: NODE_ENV,
-        fontUrl: WEBFONT_URL,
-        icons: {
-          // TODO: Move to new assset pipeline, this adds the CDN for images.
-          favicon: res.locals.asset("/images/favicon.ico"),
-          faviconSVG: res.locals.asset("/images/favicon.svg"),
-          appleTouchIcon: res.locals.asset("/images/apple-touch-icon.png"),
-        },
-        manifest: {
-          browserConfig: NOVO_MANIFEST.lookup("/images/browserconfig.xml"),
-          openSearch: NOVO_MANIFEST.lookup("/images/opensearch.xml"),
-          webmanifest: NOVO_MANIFEST.lookup("/images/manifest.webmanifest"),
-        },
-        meta: {
-          appleItunesApp: `${APP_URL}${CURRENT_PATH}`,
-        },
-        // TODO: Post-release review that sharify is still used in the template.
-        sd: sharify.data,
-      }
-
-      res.render(`${PUBLIC_DIR}/index.ejs`, options)
+      renderServerApp({ req, res, ...rest })
```

Comment:
it might be nice if this function returned a value (html) that we then handle  here, and elsewhere.